onchain/rollups/contracts/consensus/quorum/Quorum.sol

```diff
@@ -68,12 +68,16 @@ contract Quorum is AbstractConsensus, PaymentSplitter {
     /// @dev Can only be called by a validator,
     ///      and the `Quorum` contract must have ownership over
     ///      its current history contract.
-    function submitClaim(bytes calldata _claimData) external onlyValidator {
+    function submitClaim(bytes calldata _claimData) external {
         Voted storage claimYeas = yeas[_claimData];
```

Comment:
We should move this line after the if-clause, so that whenever someone (that is not a validator) calls this function, they waste less gas on a revert.